node/validator.go

```diff
@@ -58,15 +58,15 @@ type ValidatorConfig struct {
 	PrivateURL     string
 	PublicHostName string
 
-	PayAccountMode account.Mode
-	// PrivateKey private key of sentry wallet
-	PrivateKey string
-	// KeystorePath path of keystore
-	KeystorePath string
-	// PasswordFilePath stores keystore password
-	PasswordFilePath string
-	// PayAccountAddress public address of sentry wallet
-	PayAccountAddress string
+	SentryAccountMode account.Mode
+	// SentryPrivateKey private key of sentry wallet
+	SentryPrivateKey string
```

Comment:
Please dont do such modification, it is a breaking change to external validators.

If they upgrade to the latest release, then they have to change the configures.